west.yml

```diff
@@ -107,7 +107,7 @@ manifest:
       revision: d92b1bd441b41a146f04fb658fd120b70000ac73
     - name: mcuboot
       repo-path: sdk-mcuboot
-      revision: v1.9.99-ncs3
+      revision: pull/226/head
```

Comment:
Please update SHA: `20f8d86f787331daa6812a32f12077189cebd31f`

Reply:
Thank you for the SHA :)

Pushed.